### PR TITLE
Fs initdb

### PR DESCRIPTION
**Pull request by phlax**
Use pootle fs with localfs adapter for initial projects

### CODE REVIEW

File: pootle/core/initdb.py [hunk 1]
```diff
@@ -45,13 +48,13 @@ def _init_db(self, create_projects=True):
...
         self.create_root_directories()
         self.create_template_languages()
+        self.create_default_languages()
         if create_projects:
             self.create_terminology_project()
         self.create_pootle_permissions()
         self.create_pootle_permission_sets()
         if create_projects:
             self.create_default_projects()
-        self.create_default_languages()
```
Comment (unho) [inline pootle/core/initdb.py]: Moving this up looks like fixing a bug. Maybe put in a separate commit is better.

**Comment by unho** [comment]
lgtm

File: pootle/core/initdb.py (hunk 1 above)
Comment (phlax) [inline pootle/core/initdb.py]: not a bug - its necessary for the langs to be present for fs tp to sync

Comment (unho) [inline pootle/core/initdb.py]: Sorry, I was under the assumption that creating the projects would immediately trigger importing translations from disk and that therefore the TPs would only be created for the existing languages at that time.
